src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -570,28 +570,103 @@ pa.scheduler.core.authentication.loginMethod=SchedulerLDAPLoginMethod
 ----
 admin:admin
 user:pwd
 demo:demo
+guest:pwd
+test:pwd
+radmin:pwd
+nsadmin:pwd
+provider:pwd
+scheduler:scheduler_pwd
+rm:rm_pwd
+watcher:watcher_pwd
+test_executor:pwd
 ----
 
 * users membership is stored in `PROACTIVE_HOME/config/authentication/group.cfg`. For each user registered in login.cfg,
 a group membership has to be defined in this file. Each line has to look like *user:group*. Group has to be
 user to have user rights, or admin to have administrator rights. Below is an extract of `group.cfg`:
 +
-[source]
+.group.cfg
 ----
 admin:admin
+admin:nsadmins
 demo:admin
+guest:guests
+nsadmin:nsadmins
+provider:providers
+radmin:rmcoreadmins
+rm:admin
+rm:nsadmins
+scheduler:user
+test_executor:admin
 user:user
+watcher:watchers
 ----
 
 ProActive contains a set of predefined groups such as "user" and "admin". Groups are defined in `PROACTIVE_HOME/config/security.java.policy-server` as described in chapter <<User Permissions>>.
 
+In order to create new users, delete an existing user, or modify the groups or password for an existing user, a command-line tool is available.
+
+This command is available inside the *tools* folder: `PROACTIVE_HOME/tools/proactive-users`.
+
+.proactive-users
+----
+proactive-users -h
+usage: proactive-users.bat [-C | -D | -U]  [-g <GROUPS>] [-gf <GROUPFILE>] [-h]
+[-kf <KEYFILE>] [-l <LOGIN>] [-lf <LOGINFILE>] [-p
+       <PASSWORD>]
+ -C,--create                   Action to create a user
+ -D,--delete                   Action to delete an existing user
+ -g,--groups <GROUPS>          A comma-separated list of groups the user must be member of. Can be used when the user is created or
+                               updated
+ -gf,--groupfile <GROUPFILE>   Path to the group file in use [default:H:\Install\scheduling\tools\..\config/authentication/group.cfg]
+ -h,--help                     Display this help
+ -kf,--keyfile <KEYFILE>       Public key path on the local filesystem [default:H:\Install\scheduling\tools\..\config/authentication/keys/pub.key]
+ -l,--login <LOGIN>            Generate credentials for this specific user, will be asked interactively if not specified
+ -lf,--loginfile <LOGINFILE>   Path to the login file in use [default:H:\Install\scheduling\tools\..\config/authentication/login.cfg]
+ -p,--password <PASSWORD>      Password of the user, if the user is created or updated, will be asked interactively if not specified
+ -U,--update                   Action to update an existing user. Updating a user means to change the user\'s password or group
+                               membership.
+----
+
+Here are examples of use:
+
+* creating users
+
+[source]
+----
+proactive-users -C -l user1 -p pwd1 -g user
+Created user user1 in H:\Install\scheduling\tools\..\config/authentication/login.cfg
+Added group user to user user1
+----
+The user with login "user1", password "pwd1" and group "user" was created
+
+* updating users
+
+[source]
+----
+proactive-users -U -l user1 -p pwd2 -g nsadmins,admin
```

Comment:
In other examples available in the documentation, commands are prefixed by `$ `.